internals/cli/error_handler.go

```diff
@@ -0,0 +1,20 @@
+package cli
+
+import "fmt"
+
+func (c *CommandClause) argumentError(args []string) error {
+	if len(args) >= getRequired(c.Args) && len(args) <= len(c.Args) {
+		return nil
+	}
+	errorText, minimum, maximum := "", getRequired(c.Args), len(c.Args)
+
+	if minimum == maximum {
+		errorText += fmt.Sprintf("`"+c.fullCommand()+"` requires exactly %d argument(s).", minimum)
```

Comment:
To make this (and the line below) more clear, let's also include `c.fullCommand()` as a `fmt.Sprintf` parameter:
```go
errorText += fmt.Sprintf(`"%s" requires exactly %d argument(s).`, c.fullCommand(), minimum)
```

Reply:
Indeed, a nicer code. Fixed.